plugin/registry/registry.go

```diff
@@ -28,5 +29,9 @@ func GetRegistry() *plugin.Registry {
 		panic(err)
 	}
 
+	if err := r.AddSpec(trace.GetSpec()); err != nil {
+		panic(err)
+	}
+
```

Comment:
I just noticed that this file is autogenerated (I think? I don't see the first-line comment in the autogeneration code). I still think it's worth changing, but it looks like the change should be in `vbundle/templates.go`.